packages/extension/src/companion/App.tsx

```diff
@@ -44,7 +50,7 @@ export default function App({
       </style>
       <RouterContext.Provider value={router}>
         <QueryClientProvider client={queryClient}>
-          <FeaturesContext.Provider value={{ flags }}>
+          <FeaturesContext.Provider value={{ flags: memoizedFlags }}>
```

Comment:
The idea of moving to memoized flags was to not create an object on every render.
My suggestion removes the need to create the object every time.

```suggestion
          <FeaturesContext.Provider value={memoizedFlags}>
```